symplyphysics/laws/electricity/linear_charge_density_from_charge_and_length.py

```diff
@@ -0,0 +1,36 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+## Linear charge density is the quantity of charge per unit length, at any point on a line charge distribution.
+## Charge density can be either positive or negative, since electric charge can be either positive or negative.
+
+
+## Law: λ = q / l

```

Comment:
It looks like definition. Move it to definitions.
Also the commit history in your PRs keeps growing. Could you sync with main repository to cut it?